identity/terraform/main.tf

```diff
@@ -0,0 +1,41 @@
+// The service should be available at: "identity.www.wellcomecollection.org/_identity"
+
+module "identity-prod" {
+  source = "./stack"
+
+  container_image = local.prod_app_image
+  nginx_image     = local.nginx_image
+  env_suffix      = "prod"
+
+  cluster_arn  = local.prod_cluster_arn
+  namespace_id = local.prod_namespace_id
+
+  alb_listener_http_arn  = local.prod_alb_listener_http_arn
+  alb_listener_https_arn = local.prod_alb_listener_https_arn
+
+  interservice_security_group_id   = local.prod_interservice_security_group_id
+  service_egress_security_group_id = local.prod_service_egress_security_group_id
+
+  subdomain = "identity.www"
+}
+
+// The service should be available at: "identity.www-stage.wellcomecollection.org/_identity"
```

Comment:
is _identity correct?

Reply:
Just removed it - we will add the https://www-stage.wellcomecollection.org/ and root domain paths when we know them from https://github.com/wellcomecollection/platform/issues/4981